apps/src/studio/levels.js

```diff
@@ -252,6 +252,11 @@ levels.playlab_1 = utils.extend(levels.dog_hello, {
   ]
 });
 
+levels.iceage_1 = utils.extend(levels.playlab_1, {
+  background: 'icy',
+  firstSpriteIndex: 0, // manny
```

Comment:
I think they should all be an enum - it would be much easier once we have a mapping of assets. I'll add it to the todos.